modules/utils/src/signal_handler.cpp

```diff
@@ -33,6 +35,7 @@ auto TerminationBlocker::instance() -> TerminationBlocker& {
 
 auto TerminationBlocker::signalHandler(int /*unused*/) -> void {
   instance().stop_future_ = instance().app_stop_callback_();
+  LOG(INFO) << "Stop requested.";
```

Comment:
Do we have a strategy for verbose logging in hephaestus? I feel VLOG will come back to us a few more times in the future.

Reply:
I would just use VLOG for that, but typically VLOG is rather for stuff that is logged very often, not for one-off logs like this one...